content/source/docs/enterprise/api/_template.md

```diff
@@ -8,6 +8,11 @@ Follow this template to format each API method. There are usually multiple secti
 
 <!-- ^ The method and path are styled as a single code span, with global prefix (`/api/v2`) omitted and the method capitalized. -->
 
+- On success: HTTP 200 and a JSON API document (`type: "somethings"`).
```

Comment:
In keeping with the other bits I think a table here is more concise and works better when we might have multiple success and failure scenarios each with their own status code. I usually just look up a status code in the API documentation if I'm receiving it from an endpoint.

Reply:
hmmmmm in that case I'm inclined to move it down below the url parameter list. yeah?

Reply:
`Status` and `Response` would tie it in to the terminology we currently use (`Sample Response`).

Reply:
👍🏼